### ISSUE TITLE
Unable to upload scenarios to Typhoon

### ISSUE THREAD

**Issue by kabdelrahman** [zalando/typhoon#82]
* I have uploaded a scenario `new_scenario.erl` to v8.0 but when going to `http://localhost:8080/scenario/new_scenario` it's displaying an empty page.

To reproduce you can do the following:
1. Create a scenario and execute the curl command `curl -XPUT -H 'Content-Type: application/erlang' --data-binary @cs_virtual_user.erl http://localhost:8080/scenario/cs_virtual_user -v`
2. Execute curl command for typhoon scenario `curl http://localhost:8080/scenario/cs_virtual_user -vvv` 
It should display
```
*   Trying ::1...
* TCP_NODELAY set
* Connected to localhost (::1) port 8080 (#0)
> GET /scenario/cs_virtual_user HTTP/1.1
> Host: localhost:8080
> User-Agent: curl/7.51.0
> Accept: */*
> 
* Curl_http_done: called premature == 0
* Empty reply from server
* Connection #0 to host localhost left intact
curl: (52) Empty reply from server
```

**Comment by fogfish** [comment]
Thank you for the report. The issue is reproducible... Fix on the way...

**Comment by fogfish** [comment]
fixed. available at 0.8.1 release

**Comment by kabdelrahman** [comment]
Thanks, I will definitely check it.
